app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -24,11 +24,18 @@ def plan(job_invocation, host, template_invocation, proxy)
 
         raise _('Could not use any template used in the job invocation') if template_invocation.blank?
 
-        settings = { :global_proxy   => 'remote_execution_global_proxy',
-                     :fallback_proxy => 'remote_execution_fallback_proxy' }
+        if proxy.blank?
+          offline_count = options[:offline_count] || 0
+          raise n_('The only usable proxy is down',
+                   'All %{number} usable proxies are down.',
+                   offline_count) % { :number => offline_count } if offline_count > 0
```

Comment:
So, I was thinking about this, and I think we might make the error message more helpful if we told the user which proxies we tried to contact.  I also don't really like 'usable' either, maybe 'applicable' is better?

What do you think about this? "No applicable proxy could be contacted.  The follow proxies are down: %{offline}"